Monika After Story/game/script-topics.rpy

```diff
@@ -14271,3 +14271,32 @@ label monika_hot_springs:
     m 5dkbfb "Ahhh,{w=0.2} {nw}"
     extend 5dkbfa "just the thought of it makes me feel all tingly, [player]~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_dying_same_day",
+            category=['philosophy', 'romance'],
+            prompt="Dying the same day",
+            random=True,
+            aff_range=(mas_aff.ENAMORED, None),
+            conditional="renpy.seen_label('monika_back_ups')",
+        )
+    )
+
+label monika_dying_same_day:
+    m 1eua "[player], have you ever heard of some couples wishing to die the same day?"
+    m 3lksdla "Oh gosh, that sure does sound really dark, ahaha."
+    m 3wud "If I never got to meet you, I would think this was completely absurd!"
+    m 1etc "Say, why would one entirely give up on living a long and happy life, just because their partner's life dramatically cut short?"
```

Comment:
```suggestion
    m 1etc "Why would one entirely give up on living a long and happy life, just because their partner's life was dramatically cut short?"
```
This isn't really a question you'd start with `Say`.
Having `cut` in the active voice technically works, but, again, rigid.